### DISCUSSION TITLE
How do I report someone on discrord?

### DISCUSSION THREAD

**Discussion by t1devzeus** [community/community#140614] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

Please help me

**Comment by t1devzeus** [comment, marked as answer]
please!

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/140615#discussioncomment-10847082) for reference to why this discussion was closed and locked. Thanks!
